PrebidMobile/PrebidMobile-rendering/src/main/java/org/prebid/mobile/rendering/sdk/PrebidRenderingSettings.java

```diff
@@ -75,6 +73,7 @@
 
     private static SdkInitListener sInitSdkListener;
 
+    private static String sBidServerHost = "https://prebid.openx.net/openrtb2/auction";
```

Comment:
Need to remove this URL. OpenX is a custom server for Prebid. Can we mimic the host initialization behaviour from Prebid Core?

Reply:
Will do